spec/support/shared/shopify_request.rb

```diff
@@ -14,18 +14,17 @@
     ShopifyAPI::Base.user = nil
   end
 
-  def mock_request(endpoint, extension)
+  def mock_request(file, endpoint, extension)
     url = "#{ShopifyAPI::Base.site}/#{endpoint}.#{extension}"
-    json = read_file_from_endpoint(endpoint, extension)
+    json = read_file(file, extension)
```

Comment:
This is implementation (somewhat private) and this change is cool. Calling the mock request will use this underlying item and it will read from a file. All good here 👍